packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -193,6 +193,25 @@ export function computeResourceEntryProtocol(entry: RumPerformanceResourceTiming
   return entry.nextHopProtocol === '' ? undefined : entry.nextHopProtocol
 }
 
+/**
+ * Handles the 'deliveryType' property to distinguish between supported values ('cache', 'navigational-prefetch'),
+ * undefined (unsupported in some browsers), and other cases ('-' for unknown or unrecognized values).
+ * https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/deliveryType
+ */
+export function computeResourceEntryDeliveryType(entry: RumPerformanceResourceTiming): deliveryType | undefined {
+  const deliveryType = entry.deliveryType
+
+  if (deliveryType === undefined) {
+    return undefined
+  }
+
+  if (deliveryType === 'cache' || deliveryType === 'navigational-prefetch') {
+    return deliveryType
+  }
+
+  return '-'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Why using `'-'` instead of the standard `''` ?

Reply:
To indicate the difference between, we don't have the information and undefined(browser doesn't support it). We want to differentiate both cases and not just let empty for both

Reply:
ok, but Chrome is already differentiating `undefined` with `''`. Why don't we keep `''`?

Any other place we're using `'-'`?

Reply:
We discussed about it, and having a blank field is not very user friendly, as they won't be able to distinguish between an empty string and a missing field in some cases (ex: when showing that field in a column). This is why I suggested `'-'`. An alternative could be `'other'`, maybe it could be more explicit.

Reply:
👍 makes sense

Another alternative could be `'n/a'` for _not available_.